src/Kunstmaan/RedirectBundle/Resources/config/services.yml

```
@@ -22,8 +22,14 @@ services:
             - { name: form.type, alias: kunstmaan_redirect_form_type }
 
     Kunstmaan\RedirectBundle\Controller\RedirectAdminListController:
-        arguments: ['@kunstmaan_admin.domain_configuration']
+        arguments: ['@Kunstmaan\RedirectBundle\AdminList\RedirectAdminListConfigurator']
         calls:
             - setContainer: [ ] # Container service locator will be injected filled by the services from subscribed services
         tags: ['controller.service_arguments', 'container.service_subscriber']
 
+    Kunstmaan\RedirectBundle\AdminList\RedirectAdminListConfigurator:
+        arguments:
+            $em: '@doctrine.orm.entity_manager'
+            $domainConfiguration: '@kunstmaan_admin.domain_configuration'
+        autowire: true
+        autoconfigure: true
```

```suggestion
            - '@doctrine.orm.entity_manager'
            - '@kunstmaan_admin.domain_configuration'
```

Won't use autowiring/autoconfigure in bundles as the service config should be explicit, also don't use named arguments as the variables names are not included in the bc policy

---

@acrobat Done